amundsen_application/static/images/favicons/dev/browserconfig.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig>
+    <msapplication>
+        <tile>
+            <square150x150logo src="/static/images/favicons/dev/mstile-150x150.png"/>
+            <TileColor>#2d89ef</TileColor>
```

Comment:
What does this file end up achieving? Also why `#2d89ef`? This color isn't part of any of our themes.

Reply:
This one is for tiles on windows machines.
Yeah, it was par of a given set they give on the icon generator. It is not a big deal, I picked one that didn't look bad.

Reply:
Are we able to manually update these to match our theme for that bit of extra polish, or does the generator not allow it? I won't suggest any alternative colors right now without knowing what this looks like, but curious for the future.

Reply:
This generator didn't let me add the colors in this one, but gave me a bunch of options (10 or so).
We can always override it here!